packages/jest-resolve/src/resolver.ts

```diff
@@ -484,30 +488,9 @@ export default class Resolver {
     );
   }
 
-  getMockModule(from: string, name: string): string | null {
+  getMockModule(name: string): string | null {
```

Comment:
This is a breaking change, right? I mean, Jest packages can be used without Jest. So this and removal of `getMockModuleAsync()` method below are breaking. Looks like good refactor, but has to wait for major release.

Reply:
Thank you @mrazauskas. You're right, it's a breaking change for the package. Should I split the refactor into a separate PR or leave this as is for the next release?

Reply:
Sounds better to split the refactor so that the fix could land sooner.